files/ansible/deploy_benchmarks.yml

```diff
@@ -0,0 +1,67 @@
+---
+- hosts: target
+  tasks:
+        - name: Install dependencies on any Linux system
+          package:
+            name: "{{ item }}"
+            state: present
+          with_items:
+              - python3
+              - python3-pip
+              - git
+              - build-essential
+              - unzip
+              - lhasa
+
+        - name: Try to install perf, using Debian package name
+          package:
+                  name: linux-perf
+                  state: present
+          register: perf_installed
+          ignore_errors: True
+
+        - name: Try to call perf, to see if it's installed at all
```

Comment:
This is a little backwards. You should test first, then try to install the package if not there.

Also, this seems true for all packages, if we want to be start testing on systems that run Ubuntu, CentOS, OpenSuse, Arch... we either keep multiple package names lists, or we try to run it, and if we can't install, fail and ask the users to install it themselves.

Reply:
I added a task 'install_perf' that should be able to do the job for OpenSUSE, Arch Linux, CentOS, Ubuntu and Debian (it might work with RHEL as well).